.github/ISSUE_TEMPLATE/pre-release.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Pre-Release Operator Rust Updates
+about: This template can be used to track the progress of Rust updates across our operators leading up to the Stackable release
+title: "chore(tracking): Update Rust dependencies of operators for Stackable release XX.(X)X"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
+
+## Pre-Release Operator Rust Updates
+
+<!--
+    Replace 'TRACKING_ISSUE' with the applicable release tracking issue number.
+-->
+
+Part of <https://github.com/stackabletech/issues/TRACKING_ISSUE>
+
+Replace the items in the task lists below with the applicable Pull Requests
+
+> [!TIP]
+> Use the links below to automatically create applicable PRs for each operator
+> using the PR template.
+
+<!--
+    The following list was generated by:
+
+    cd operator-templating
+    yq '.repositories[].name' config/repositories.yaml \
+    | sort \
+    | xargs -I {} echo "- [ ] https://github.com/stackabletech/{}/compare/main..chore/bump-rust-deps?quick_pull=1&template=pre-release.md&title=chore%3A+Bump+Rust+dependencies"
```

Comment:
Maybe we should suggest the branch name to use?
Also, should we suggest adding the release version number to it to avoid issues where people have the branch locally from the previous release (if they don't prune)?

I know it's easy to update the links below, but even better if they don't have to.

```suggestion
    # go to the stackable-templating repository, then run:
    yq '.repositories[].name' config/repositories.yaml \
    | sort \
    | xargs -I {} echo "- [ ] https://github.com/stackabletech/{}/compare/main..chore/bump-rust-deps-pre-$(date +%y.%-m)?quick_pull=1&template=pre-release.md&title=chore%3A+Bump+Rust+dependencies+pre-$(date +%y.%-m)"
```